modin/pandas/concat.py

```diff
@@ -132,3 +139,25 @@ def concat(
         else:
             result_df.columns = new_idx
     return result_df
+
+
+def _determine_name(objs: list, axis):
+    axis_getter = (lambda df: df.columns) if axis else (lambda df: df.index)
+    name_getter = (
+        (lambda df: axis_getter(df).names)
```

Comment:
```suggestion
        (lambda df: df.axes[axes].names)
```

Reply:
we can't use this, because `_determine_name` can obtain a list of query compilers as `objs`, but query compiler does not have field `axes`